minidump-processor/src/stackwalker/arm64.rs

```diff
@@ -429,6 +432,12 @@ impl Unwind for ArmContext {
     {
         let stack = stack_memory.as_ref()?;
 
+        let last_sp = callee.context.get_stack_pointer();
+
+        if last_sp == 0x0000000176fc2fd0 {
+            println!("Here!");
+        }
+
```

Comment:
Leftover debugging code.

```suggestion
        let last_sp = callee.context.get_stack_pointer();

        if last_sp == 0x0000000176fc2fd0 {
            println!("Here!");
        }

```